scripts/install

```diff
@@ -76,16 +62,9 @@ ynh_script_progression --message="Configuring permissions..." --weight=2
 if [ "$redirect_type" != "private_proxy" ]
 then
 	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_permission_update --permission="main" --add="visitors"
+    ynh_permission_update --permission="main" --add="visitors"
 fi
```

Comment:
hmpf, what do with this ... the proxy type and the ACL should be two different things ..